src/js/options.js

```diff
@@ -737,6 +765,40 @@ function toggleWebRTCIPProtection() {
   });
 }
 
+// handles toggling the alternateErrorPagesEnabled setting
+function toggleAlternateErrorPagesSetting() {
+  // ensure this is only attempting to be set on supportive browsers
+  if (!OPTIONS_DATA.alternateErrorPagesAvailable) {
+    return;
+  }
+
+  let cps = chrome.privacy.services;
+
+  // whatever the current setting is at, reverse it
```

Comment:
We should instead `clear({})` the setting. It doesn't make sense for us to continue to override the setting (to be enabled) if the user disables our override for whatever reason. We should relinquish control instead.

Reply:
I feel like going this route could be misleading to the user.

I'm not sure there's a way to describe this to the user in the options page. For instance, when control is relinquished and the browser preference is cleared, it doesn't necessarily mean that the value is set to `true`. I think it would take disabling the checkbox or adding more info tooltip text to try and describe this situation... which I think we are trying to move away from overcomplicating, right? If the user decides to override these settings elsewhere, in the browser or with some other browser extension, the current working state of this feature will update the options page checkbox to show how the preference is currently set (and toggling it would update the settings storage to its current state if in that case there is some discrepancy).

Reply:
Our privacy overrides take the form of "Disable INVASIVE_SETTING". If the user disables an override, they tell Privacy Badger to stop disabling INVASIVE_SETTING. I think it's clear the checkbox doesn't ever enable INVASIVE_SETTING.